searx/templates/simple/categories.html

```diff
@@ -12,7 +12,7 @@
     'social media': 'people-outline',
 }  -%}
 <div id="categories" class="search_categories">{{- '' -}}
-    <div id="categories_container">
+    <div id="categories_container" class="nojs">
```

Comment:
I'm not sure if it can replace this line change, but there is `ns-js` or `js` class on the `<html>` element:

https://github.com/searxng/searxng/blob/342d321196a84437d51c75da348c533e01626361/searx/templates/simple/base.html#L2

https://github.com/searxng/searxng/blob/342d321196a84437d51c75da348c533e01626361/searx/static/themes/simple/src/js/head/00_init.js#L17-L18

Reply:
Indeed we can, thank you Alexandre!